src/sentry/static/sentry/app/components/events/interfaces/spans.tsx

```diff
@@ -0,0 +1,467 @@
+  }) => {
+    const spanChildren: SpanType[] = _.get(lookup, spanID, []);
+
+    const renderedSpanChildren = spanChildren.map(span => {
+      const key = `${traceID}${span.span_id}`;
+
+      // if (
+      //   start_timestamp === undefined ||
+      //   (_.has(span, 'start_timestamp') &&
+      //     _.isNumber(span.start_timestamp) &&
+      //     span.start_timestamp < start_timestamp)
+      // ) {
+      //   start_timestamp = span.start_timestamp;
+      // }
+
+      // if (
+      //   !end_timestamp ||
+      //   (_.has(span, 'timestamp') &&
+      //     _.isNumber(span.timestamp) &&
+      //     span.timestamp > end_timestamp)
+      // ) {
+      //   end_timestamp = span.timestamp;
+      // }
+
+      return (
+        <React.Fragment key={key}>
+          {this.renderSpan({
+            span,
+            spanID: span.span_id,
+            traceID,
+            lookup,
+            generateBounds,
+          })}
+        </React.Fragment>
+      );
+    });
+
+    return (
+      <React.Fragment>
+        <Span span={span} generateBounds={generateBounds} />
+        {renderedSpanChildren}
+      </React.Fragment>
+    );
+  };
+
+  renderRootSpan = () => {
+    const {event} = this.props;
+
+    const trace: TraceType | undefined = _.get(event, 'contexts.trace');
+
+    if (!trace) {
+      return null;
+    }
+
+    const parsedTrace = this.parseTrace();
+
+    // TODO: ideally this should be provided
+    const rootSpan: SpanType = {
+      trace_id: trace.trace_id,
+      parent_span_id: void 0,
+      span_id: trace.span_id,
+      start_timestamp: parsedTrace.traceStartTimestamp,
+      timestamp: parsedTrace.traceEndTimestamp,
+      same_process_as_parent: true,
+      op: void 0,
+      data: {},
+    };
+
+    const traceEndTimestamp = _.isNumber(parsedTrace.traceEndTimestamp)
+      ? parsedTrace.traceStartTimestamp == parsedTrace.traceEndTimestamp
+        ? parsedTrace.traceStartTimestamp + 0.05
+        : parsedTrace.traceEndTimestamp
+      : parsedTrace.traceStartTimestamp + 0.05;
+
+    const generateBounds = boundsGenerator({
+      traceStartTimestamp: parsedTrace.traceStartTimestamp,
+      traceEndTimestamp: traceEndTimestamp,
+      viewStart: 0,
+      viewEnd: 1,
+    });
+
+    return this.renderSpan({
+      span: rootSpan,
+      spanID: trace.span_id,
+      traceID: trace.trace_id,
+      lookup: parsedTrace.lookup,
+      generateBounds,
+    });
+  };
+
+  renderMinimap = () => {
+    if (!this.state.renderMinimap) {
+      return null;
+    }
+
+    return <TraceViewMinimap traceViewRef={this.traceViewRef} />;
+  };
+
+  componentDidMount() {
+    if (this.traceViewRef.current) {
+      this.setState({
+        renderMinimap: true,
+      });
+    }
+  }
+
+  render() {
+    const {event, type} = this.props;
+
+    // console.log('event', event);
+
+    const title = (
+      <div>
+        <GuideAnchor target="spans" position="top">
```

Comment:
Do we have guides for spans?

Reply:
@markstory I'm unsure what you meant. Do you mean guide anchors for the spans?

Reply:
These lines of code are now removed.